cmd/kubeadm/app/apis/kubeadm/v1alpha1/defaults.go

```diff
@@ -26,14 +26,12 @@ import (
 const (
 	DefaultServiceDNSDomain  = "cluster.local"
 	DefaultServicesSubnet    = "10.96.0.0/12"
-	DefaultKubernetesVersion = "latest-1.6"
-	// This is only for clusters without internet, were the latest stable version can't be determined
-	DefaultKubernetesFallbackVersion = "v1.6.0"
-	DefaultAPIBindPort               = 6443
-	DefaultDiscoveryBindPort         = 9898
-	DefaultAuthorizationMode         = "RBAC"
-	DefaultCACertPath                = "/etc/kubernetes/pki/ca.crt"
-	DefaultCertificatesDir           = "/etc/kubernetes/pki"
+	DefaultKubernetesVersion = "stable"
```

Comment:
this means kubeadm 1.6.2ish would pick up k8s 1.7 when it's released? is that what we want?